packages/block-library/src/comments-pagination/editor.scss

```diff
@@ -0,0 +1,38 @@
+$pagination-margin: 0.5em;
+
+// Center flex items. This has an equivalent in style.scss.
+.wp-block[data-align="center"] > .wp-block-comments-pagination {
+	justify-content: center;
+}
+
+.editor-styles-wrapper {
+	.wp-block-comments-pagination {
+		max-width: 100%;
+		&.block-editor-block-list__layout {
+			margin: 0;
+		}
+	}
+}
+
+.block-library-comments-pagination-toolbar__popover .components-popover__content {
```

Comment:
Where is it coming from? The class name isn’t prefixed with `wp-` so it raises a flag, but it might be as well some legacy name we need to maintain.

Reply:
That CSS class is the same that the [Query Pagination block has](https://github.com/WordPress/gutenberg/blob/6f3eb95d6f5eef0ff5a21ea5b2cf85ffba124e3d/packages/block-library/src/query-pagination/editor.scss#L17) ― just changing `query` by `comments-pagination` (potential bad copy-paste? 👀 ).

By the way, I realized that the class name `block-library-query-toolbar__popover` is being used inside the [`QueryToolbar` component](https://github.com/WordPress/gutenberg/blob/bbaeea2056c96af931170da1665d5d98c79b4b8d/packages/block-library/src/query/edit/query-toolbar.js#L47) (in the Query block), but the equivalent component for the Comments Query Loop block ― i.e. [`CommentsQueryLoopToolbar`](https://github.com/WordPress/gutenberg/blob/d51ac8a86b1756da64840b02f2ab5c129443391a/packages/block-library/src/comments-query-loop/toolbar.js) ― doesn't use any class.

I'm looking at whether it makes sense to add that style or not.

Reply:
OK, after taking a deeper look I don't see any reason to keep that style.

In fact, it seems we could also safely remove the [same code](https://github.com/WordPress/gutenberg/blob/6f3eb95d6f5eef0ff5a21ea5b2cf85ffba124e3d/packages/block-library/src/query-pagination/editor.scss#L17-L19) from the Query Pagination block as well, right? There is no element using that class (if I'm not mistaken).

That class actually looks like a copy of [this one](https://github.com/WordPress/gutenberg/blob/6244cf57555665dd55d64caad98674848f73678a/packages/block-library/src/query/editor.scss#L1-L3) from the Query block, which styles the [`QueryToolbar` component](https://github.com/WordPress/gutenberg/blob/bbaeea2056c96af931170da1665d5d98c79b4b8d/packages/block-library/src/query/edit/query-toolbar.js#L47) I mentioned previously.